### PR TITLE
[NPU][LLM] add README & reformat llama scripts

### PR DESCRIPTION
**Pull request by SylarTiaNII**
### PR types

Others
### PR changes

Others
### Description

add README for llama training/inference on NPU(910)

### CODE REVIEW

**Review by ronny1996 — APPROVED** [review]
LGTM

**Comment by onecatcn** [comment]
LGTM

File: llm/llama/npu/README.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+## 🚣‍♂️ 使用PaddleNLP在NPU下跑通llama2-13b模型 🚣
+PaddleNLP在昇腾NPU（[了解昇腾](https://www.hiascend.com/zh/ecosystem/industry)）上对llama2-13B模型进行了深度适配和优化，该套件实现了昇腾NPU和GPU的训推入口基本统一，达到了『无缝切换』的效果。
```
Comment (ZHUI) [inline llm/llama/npu/README.md]: https://github.com/PaddlePaddle/PaddleNLP/tree/develop/llm/npu/llama

文件放到 llm/npu 下面，我们重构的pr已经合入了

File: llm/llama/npu/README.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  "rope_scaling": null,
+  "tie_word_embeddings": false,
+  "torch_dtype": "float16",
+  "transformers_version": "4.31.0",
+  "use_cache": false,
+  "vocab_size": 32000
+}
+```
+为了保障极致压缩的推理成本，我们使用了静态图实现。因此需要从训练产出的动态图模型中导出静态图模型，执行如下命令进行导出：
+```
```
Comment (ronny1996) [inline llm/llama/npu/README.md]: 改为
```
cd PaddleNLP/llm
python predict/export_model.py --model_name_or_path merged_model  --inference_model --output_path ./inference --dtype float16  --device npu  --block_attn
```
当前看必须在 PaddleNLP/llm 才跑得通

File: llm/llama/npu/README.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+python export_model.py --model_name_or_path merged_model  --inference_model --output_path ./inference --dtype float16  --device npu  --block_attn
+```
+最终，我们通过静态图的模型执行推理：
+```
+# 执行推理代码
```
Comment (ronny1996) [inline llm/llama/npu/README.md]: 改为 
```
# 执行推理代码
python predict/predictor.py  --model_name_or_path ./inference --inference_model --dtype "float16" --mode "static" --block_attn --device npu
```

**Review by ronny1996 — CHANGES_REQUESTED** [review]

**Comment by ronny1996** [comment]
https://github.com/PaddlePaddle/PaddleNLP/blob/develop/llm/predict/export_model.py#L104 这里帮忙改成 
```
from npu.llama.export_utils import process_params
```

File: llm/export_npu.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+set -x
+
+src_path=${1:-".npu/llama/output/sft_bf16_llama_N1C8/"}
+dst_path=${2:-"./inference"}
+
+source /usr/local/Ascend/ascend-toolkit/set_env.sh
+source /usr/local/Ascend/atb/set_env.sh
+
+export PYTHONPATH=../:$PYTHONPATH
```
Comment (ZHUI) [inline llm/export_npu.sh]: 刚到 llm/npu

File: llm/predict_npu.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+model_path=${1:-"./inference"}
+
+source /usr/local/Ascend/ascend-toolkit/set_env.sh
```
Comment (ZHUI) [inline llm/predict_npu.sh]: 放到llm/npu

**Review by ZHUI — CHANGES_REQUESTED** [review]

**Review by ZHUI — APPROVED** [review]

**Pull request merged**